front-end/app/templates/index.hbs

```diff
@@ -1,3 +1,13 @@
-{{page-title "FrontEnd"}}
-<NavBar/>
+{{page-title 'FrontEnd'}}
+<NavBar />
+
+<p>{{this.clickedPosition}}</p>
+<MapComp
+  @lat={{lat}}
+  @lng={{lng}}
+  @zoom={{zoom}}
+  @markers={{this.markers}}
+  @onMapClick={{this.setClickedPosition}}
+/>
+
 {{outlet}}
```

Comment:
The outlet tag is used so that it's display on every page. It's kinda an global override, so I'd suggest removing it, otherwise it will show up on every page.
```suggestion

```